server/app/services/program/ProgramService.java

```diff
@@ -656,7 +664,7 @@ public ProgramDefinition newDraftOf(long id) throws ProgramNotFoundException {
 
   private ImmutableSet<CiviFormError> validateProgramData(
       String displayName,
-      String displayDescription,
+      String shortDescription,
```

Comment:
short description is required and long description (aka the former description field) is now optional, so we no longer need to pass it through validation.